src/odemis/gui/comp/stream.py

```diff
@@ -125,7 +125,7 @@ def __init__(self, parent, wid=wx.ID_ANY, pos=wx.DefaultPosition, size=wx.Defaul
         self.btn_update = self._add_update_btn() if self.Parent.options & OPT_BTN_UPDATE else None
 
         # The spacer is responsible for creating padding on the right side of the header panel
-        self._sz.AddSpacer((64, 16))
+        self._sz.AddSpacer(64)#todo: 16 left in vertical
```

Comment:
This probably can be replaced by `self._sz.Add(64, 16, 0)`.

Reply:
It does not. Add accepts a single argument.

Reply:
It also does not accept a tuple like it previously did.

Reply:
If you look at the documentation, you can see that `Add()` accepts multiple arguments. Espcially, the documentation of `AddSpacer()` explains how to add a spacer with `Add()`: https://wxpython.org/Phoenix/docs/html/wx.Sizer.html#wx.Sizer.AddSpacer

Anyway, I've looked further at the code, and the 16 px vertical is not needed anyway. So you can leave the line as-is and drop the `TODO` comment.

Reply:
Oh, right, we confused AddSpacer and Add methods. Ok, todo dropped, changes pushed.